nix/sources.json

```diff
@@ -11,16 +11,28 @@
         "url": "https://github.com/yihuang/gomod2nix/archive/af2ff6eb58f1ebbc374bc2fa0c23fa105b2d490f.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
+    "ibc-rs": {
+        "branch": "allthatjazzleo/coin-type",
```

Comment:
they have merged to master
https://github.com/informalsystems/ibc-rs/commit/c32393a4f344877e34957d13a768e6623486c587
maybe use master branch?

Reply:
done